packages/react-components/react-aria/library/src/AriaLiveAnnouncer/useAriaNotifyAnnounce.ts

```diff
@@ -33,9 +31,7 @@ export const useAriaNotifyAnnounce_unstable = (): AriaLiveAnnounceFn => {
 
       // map fluent announce options to ariaNotify options
       const ariaNotifyOptions: AriaNotifyOptions = {
-        notificationID: batchId,
-        priority: priority > 1 ? 'important' : 'none',
-        interrupt: batchId ? (priority > 0 ? 'all' : 'pending') : 'none',
+        priority: priority > 1 ? 'high' : 'normal',
```

Comment:
Wouldn't this break in older browser versions that expect "important" and "none"?

Reply:
those browser versions won't expose `ariaNotify` to general users, so it should be largely unimportant -- you'd have to have both gone into browser settings to turn on this specific flag, and not have updated your browser (and I'm fine saying that users who turn on esoteric browser flags are responsible for when they break and need an update)

Reply:
Cool, thanks!